src/algebra/group_power/basic.lean

```diff
@@ -612,6 +612,26 @@ theorem pow_bit0_pos {a : R} (h : a ≠ 0) (n : ℕ) : 0 < a ^ bit0 n :=
 theorem pow_two_pos_of_ne_zero (a : R) (h : a ≠ 0) : 0 < a ^ 2 :=
 pow_bit0_pos h 1
 
+variables {x y : R}
+
+@[simp] theorem sqr_abs : abs x ^ 2 = x ^ 2 :=
```

Comment:
I think `x ^ 2` is usually referred to as `sq`, not `sqr`: https://leanprover-community.github.io/mathlib_docs/algebra/group_power/basic.html#sq_sub_sq

Also, it looks like this lemma may have been added recently: https://leanprover-community.github.io/mathlib_docs/algebra/group_power/basic.html#abs_sq_eq

Reply:
I was going off of the naming in `data.real.sqrt`, which contains theorems named `sqr_sqr`, `sqrt_sqr`, `sqrt_eq_iff_sqr_eq`, `sqrt_sqr_eq_abs`, etc.

Reply:
Ah, OK. You can keep the names as is, but we should make them consistent in a followup.

Reply:
No problem, I can help take care of that if there's a consensus regarding which to keep.

Reply:
Regarding your second point, that is the same as `abs_sqr` here, not `sqr_abs`, correct?

Reply:
My preference would be for `sq` since it's shorter, but it's probably making a poll on Zulip.

Reply:
I agree; in fact, I was surprised to find it called `sqr` and not `sq`. It's also referred to as `pow_two` in some places, e.g. `pow_two_nonneg`.

Reply:
I'll merge master to deal with the now redundant lemma.